examples/others/controller/lv_example_controller_1.c

```diff
@@ -0,0 +1,42 @@
+/**
+ * @file lv_example_controller_1.c
+ * @brief Basic usage of obj controller
+ */
+#include "../../lv_examples.h"
+
+#if LV_USE_OBJ_CONTROLLER && LV_BUILD_EXAMPLES
+
+static void sample_controller_ctor(lv_obj_controller_t *self, void *args);
+
+static lv_obj_t *sample_controller_create_obj(lv_obj_controller_t *self, lv_obj_t *parent);
+
+struct sample_controller_t {
+    lv_obj_controller_t base;
+    const char *counter;
```

Comment:
Why is it called `counter`? `title` or so seems more reasonable to me.

Reply:
Changed.